code/modules/surgery/organs/lungs.dm

```diff
@@ -269,25 +271,39 @@
 		species_traits = H.species.species_traits
 
 	if(!(COLDRES in H.mutations) && !(RESISTCOLD in species_traits)) // COLD DAMAGE
-		var/cold_modifier = H.species.coldmod
```

Comment:
why not just take the absolute value of the species' cold mod and be done with it? Why the extra `cold_modifier` and `heat_modifier`?